README.rst

```diff
@@ -156,15 +156,18 @@ type definitions:
 Known implementations
 ~~~~~~~~~~~~~~~~~~~~~
 
-- in Golang: https://github.com/package-url/packageurl-go
 - for .NET: https://github.com/package-url/packageurl-dotnet
-- for the JVM: https://github.com/package-url/packageurl-java,
+- in Elixir: https://github.com/jshmrtn/purl
```

Comment:
Commit message: This is not only adding new known implementations, but also performs reordering and rewording the titles of existing ones, making the change harder to review.